### PR TITLE
added backdrop clientOption for prevent modal close if click outside of it

### PR DESCRIPTION
**Pull request by paskuale75**

### CODE REVIEW

File: assets/modal-form.js [hunk 1]
```diff
@@ -50,6 +50,11 @@
             } else if (clientOptions.tabindex == false) {
                 $modalDialog.removeAttr('tabindex');
             }
+            if (clientOptions.backdrop) {
+                $modalDialog.attr('data-backdrop', clientOptions.backdrop);
+            } else if (clientOptions.backdrop == false) {
```
Comment (borodulin) [inline assets/modal-form.js:55]: clientOptions.backdrop === false 
Is strict comparison required?

Comment (paskuale75) [reply assets/modal-form.js:55]: You are right, it's not required

Comment (borodulin) [reply assets/modal-form.js:55]: So, if (clientOptions.backdrop == false) - this comparison is also not required

Comment (paskuale75) [reply assets/modal-form.js:55]: so I change the code and raise the pull request?

**Pull request merged**